### PR TITLE
Add support for empty <style/> tags

### PR DESCRIPTION
**Pull request by iherasymenko**

### CODE REVIEW

**Comment by asturur** [comment]
Seems good to me.
Would you please remove the IE9 comment and the relative code? ie9 is no more supported.

**Comment by iherasymenko** [comment]
Sure thing. Done!

**Comment by asturur** [comment]
sorry could i ask you a test for loadSVGFromString that will keep this feature working from regression? A unit test should be enough. sorry for not asking before

**Comment by iherasymenko** [comment]
No problem at all. I've just pushed the changes. I added a unit test for the `getCssRule` method.

Please let me know if you indeed want me to a test case for `loadSVGFromString`. 

Thanks!

**Comment by iherasymenko** [comment]
Sorry, the test failing in Chrome and Firefox; works fine on Node though. I'd appreciate if you could tell me how to see the stacktrace of the failing test.

File: test/unit/parser.js [hunk 1]
```diff
@@ -737,6 +737,19 @@
...
+
+    var doc = fabric.document,
+        styleElement = doc.createElement('style');
+    doc.body.innerHTML = '';
+    doc.body.appendChild(styleElement);
+
+    var expectedObject = {};
+
+    assert.deepEqual(fabric.getCSSRules(doc), expectedObject);
+  });
```
Comment (asturur) [inline test/unit/parser.js]: i think copying this test here:
https://github.com/fabricjs/fabric.js/pull/6169/files#diff-2435ff7fbf121d1b599209a2fedea94dR755
Adding an empty style tag in the  SVG string and just asserting that objects are more than 0, is enough and simpler for you. What do you think? i m sorry i could not get back sooner

**Comment by iherasymenko** [comment]
@asturur done. The CI pipeline is now green :-)

**Comment by asturur** [comment]
perfect! less code and is functional.

**Comment by iherasymenko** [comment]
Thank you 😊

**Pull request merged**